src/bic/fix-animation.js

```diff
@@ -26,15 +27,17 @@ define(function (require) {
       !!_.intersection(classes, TRANSITIONS).length;
   }
 
-  $(window).on('animationstart', function (event) {
+  $window.on('animationstart', function (event) {
     var timer;
     if (isJQMTransition(event)) {
       // okay, we need to set a 1s timer in case this doesn't finish
       timer = setTimeout(function () {
         c.warn('"animationend" event expected but not observed');
-        $(event.target).trigger('animationend');
+        $window.trigger('animationend');
```

Comment:
Assuming the event bubbles properly, I wonder if it wouldn't be safer to still `.trigger()` the event on the original target element?